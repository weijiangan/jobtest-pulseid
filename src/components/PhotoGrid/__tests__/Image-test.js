import React from "react";
import { render, cleanup } from "@testing-library/react";
import { Image } from "../";

afterEach(cleanup);

test("matches snapshot", () => {
  const { asFragment } = render(<Image />);
  expect(asFragment()).toMatchSnapshot();
});
