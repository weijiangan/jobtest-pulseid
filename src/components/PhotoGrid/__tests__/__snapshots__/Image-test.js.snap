// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <img />
  <div
    class="imgPlaceholder"
    style="opacity: 1;"
  />
</DocumentFragment>
`;
